### PR TITLE
Add Rehau integration documentation

### PR DESCRIPTION
**Pull request by Jeoffreybauvin**
## Proposed change

Documentation for the new Rehau integration (pull request  component in progress)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch). 
- [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/34727
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1518
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Rehau Nea Smart Manager"
+description: ""
+ha_release: "0.0.1"
```
Comment (klaasnicolaas) [inline source/_integrations/rehau_nea_smart_manager.markdown]:
```suggestion
ha_release: "0.110"
```

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+This integration generates a climate entity for each heatarea in your Nea Smart Manager.
+
+To enable this platform, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: rehau
+    host: '192.168.1.2'
```
Comment (klaasnicolaas) [inline source/_integrations/rehau_nea_smart_manager.markdown]:
```suggestion
    host: IP_ADDRESS
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Preset modes
+
+| Preset mode | Description |
+| ---------------------- | -------- |
+| `auto` | Auto mode : use Nea Smart Manager scheduling
+| `eco` | Eco mode
+| `comfort` | Comfort mode
+
+## Component services
```
Comment (frenck) [inline source/_integrations/rehau_nea_smart_manager.markdown]: This section only contains services provided by the climate component. Hence, they should not be documented here as they are documented by the climate integration itself.

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Integrates [Rehau Nea Smart Manager](https://www.rehau.com/en-en/nea-smart/francais) Thermostat controller into Home Assistant.
+
+This integration generates a climate entity for each heatarea in your Nea Smart Manager.
+
+To enable this platform, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: rehau
```
Comment (frenck) [inline source/_integrations/rehau_nea_smart_manager.markdown:23]: We don't allow new platform based integrations. See ADR-0007:

<https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md>

Furthermore, this integration communicates with a device or service, so ADR-0010 also applies:

<https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+ha_quality_scale: internal
+ha_config_flow: true
+ha_codeowners:
+  - '@Jeoffreybauvin'
+ha_domain: rehau
+---
+
+Integrates [Rehau Nea Smart Manager](https://www.rehau.com/en-en/nea-smart/francais) Thermostat controller into Home Assistant.
+
+This integration generates a climate entity for each heatarea in your Nea Smart Manager.
```
Comment (frenck) [inline source/_integrations/rehau_nea_smart_manager.markdown]:
```suggestion
This integration generates a climate entity for each heating area in your Nea Smart Manager.
```

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 6]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Rehau Nea Smart Manager"
+description: ""
```
Comment (frenck) [inline source/_integrations/rehau_nea_smart_manager.markdown]: Description missing.

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+ha_release: "0.0.1"
+ha_category: Climate
+ha_iot_class: "Local Polling"
+ha_quality_scale: internal
```
Comment (frenck) [inline source/_integrations/rehau_nea_smart_manager.markdown]: This is not an internal component of Home Assistant.

File: source/_integrations/rehau_nea_smart_manager.markdown [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+ha_config_flow: true
```
Comment (frenck) [inline source/_integrations/rehau_nea_smart_manager.markdown:8]: The description below does not suggest it is using a configuration flow but YAML configuration?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/rehau_nea_smart_manager.markdown (hunk 2 above)
Comment (Jeoffreybauvin) [reply source/_integrations/rehau_nea_smart_manager.markdown]: Fixed.

File: source/_integrations/rehau_nea_smart_manager.markdown (hunk 1 above)
Comment (Jeoffreybauvin) [reply source/_integrations/rehau_nea_smart_manager.markdown]: Fixed.

File: source/_integrations/rehau_nea_smart_manager.markdown (hunk 3 above)
Comment (Jeoffreybauvin) [reply source/_integrations/rehau_nea_smart_manager.markdown]: Removed, thanks.

File: source/_integrations/rehau_nea_smart_manager.markdown (hunk 5 above)
Comment (Jeoffreybauvin) [reply source/_integrations/rehau_nea_smart_manager.markdown]: Fixed.

File: source/_integrations/rehau_nea_smart_manager.markdown (hunk 7 above)
Comment (Jeoffreybauvin) [reply source/_integrations/rehau_nea_smart_manager.markdown]: Fixed.

File: source/_integrations/rehau_nea_smart_manager.markdown (hunk 6 above)
Comment (Jeoffreybauvin) [reply source/_integrations/rehau_nea_smart_manager.markdown]: Fixed.

**Comment by frenck** [comment]
Closing PR since the parent PR has been closed.

**Pull request closed without merging**